### DISCUSSION TITLE
Can we maintain dbt-starrocks in dbt-labs?

### DISCUSSION THREAD

**Discussion by Astralidea** [dbt-labs/dbt-core#8190] [Q&A]
Hello, I am an eco-developer from Starrocks.
Starrrocks is an analytics-oriented database. https://github.com/StarRocks/starrocks
dbt is a very popular tool, and we have some customers who want Starrocks to better adapt to dbt.

My question is What can we do to make it easier for users to use dbt?
We now know that we can modify the official documentation and add the method we use, which we will continue to do later.
Can we let you create a dbt-starrocks repositorie under dbt-labs (like dbt-redshift dbt-snowflake dbt-bigquery) and then our developers will maintain this repositorie?
Or what requirements do we need to meet in order to do so? Or do you recommend in this way?

**Comment by jaypeedevlin** [comment]
Hi, I don't work for dbt, but generally the process that you would follow is to make your own adapter which you can maintain to facilitate folks using dbt with your database.

Here's a good place to start reading: https://docs.getdbt.com/guides/dbt-ecosystem/adapter-development/1-what-are-adapters
